src/aliceVision/sfm/pipeline/sequential/ReconstructionEngine_sequentialSfM.cpp

```diff
@@ -227,7 +227,16 @@ bool ReconstructionEngine_sequentialSfM::process()
 
     exportStatistics(elapsedTime);
 
-    return !_sfmData.getPoses().empty();
+    int nbviews = _sfmData.getViews().size();
+    int nbposes = _sfmData.getPoses().size();
+
+    int minPoses = 1;
+    if (nbviews > 5)
+    {
+        minPoses = 3;
+    }
+
+    return (nbposes >= minPoses);
```

Comment:
```suggestion
    const auto nbviews = _sfmData.getViews().size();
    const auto nbposes = _sfmData.getPoses().size();

    const std::size_t minPoses = nbviews > small_dataset? 3 : 1;

    return (nbposes >= minPoses);
```
where `small_dataset` is the placeholder for the new param discussed above

Reply:
ok, good idea, i'll replace.